disttask/framework/dispatcher/dispatcher_test.go

```diff
@@ -217,6 +217,9 @@ func checkDispatch(t *testing.T, taskCnt int, isSucc bool, isCancel bool) {
 			dispatcher.DefaultDispatchConcurrency = originalConcurrency
 		}
 	}()
+
+	require.NoError(t, mgr.StartManager(":4000", "background"))
```

Comment:
if this code will be complied into release version？

Reply:
No